visualization/include/pcl/visualization/area_picking_event.h

```diff
@@ -50,9 +50,14 @@ namespace pcl
     class PCL_EXPORTS AreaPickingEvent
     {
       public:
-        AreaPickingEvent (int nb_points, const std::vector<int>& indices)
+       PCL_DEPRECATED(1,11,"This constructor is deprecated!") AreaPickingEvent (int nb_points, const Indices& indices)
+        : nb_points_ (nb_points)
+        , indices_ (indices)
```

Comment:
You can delegate it like: `: AreaPickingEvent(nb_points, {"all_clouds",indices}) {}`